### ISSUE TITLE
GTS3 DMD brightness low

### ISSUE THREAD

**Issue by francisdb** [vpinball/pinmame#346]
10.8.1/ Linux standalone

Previously reported in #332 

GTS3 DMD based games have low brightness (Street Fighter II / Waterworld / Barb Wire)

Example for Street Fighter II
![image](https://github.com/user-attachments/assets/1e38e58a-b6d0-42df-b48a-9ca06fda778c)

**Comment by vbousquet** [comment]
Could you be more precise on the app/library versions ?

**Comment by francisdb** [comment]
Maybe @jsm174 can confirm this?

**Comment by jsm174** [comment]
Last nights builds of standalone and 10.8.1 are built with the latest pinmame. I don't have this table set up at the moment, but I can check.

**Comment by francisdb** [comment]
This has been the case for at least 2 weeks. Could of course have been like this for a while already.

**Comment by jsm174** [comment]
did you test with last nights builds? It has the pinmame from last night.

**Comment by francisdb** [comment]
yes, mentioned pinmame commit in the issue decription, still very dim here (and blurry since SDL3)

Barb Wire
<img width="309" alt="image" src="https://github.com/user-attachments/assets/ccab706f-2cc0-4e78-b596-230ec73383e8">

**Comment by jsm174** [comment]
@vbousquet - would I need to adjust libdmdutil?

https://github.com/vpinball/libdmdutil/blob/8e110d87edab1b843d97ba831743c79519e07ad8/include/DMDUtil/LevelDMD.h#L30-L34

**Comment by jsm174** [comment]
So I'm not quite sure how to fix this with all the latest changes to pinmame, but there used to be code like this:

https://github.com/vpinball/pinmame/blob/65cdbf58bc7be6ef414ea273d1bb8d4cb42471c8/src/wpc/core.c#L1987-L1990

I had this copied in libdmdutil, since it uses a LevelDMD:

https://github.com/vpinball/libdmdutil/blob/8e110d87edab1b843d97ba831743c79519e07ad8/include/DMDUtil/LevelDMD.h#L29-L34

```
 private:
  static constexpr uint8_t LEVELS_WPC[] = {0x14, 0x21, 0x43, 0x64};
  static constexpr uint8_t LEVELS_GTS3[] = {0x00, 0x1E, 0x23, 0x28, 0x2D, 0x32, 0x37, 0x3C,
                                            0x41, 0x46, 0x4B, 0x50, 0x55, 0x5A, 0x5F, 0x64};
  static constexpr uint8_t LEVELS_SAM[] = {0x00, 0x14, 0x19, 0x1E, 0x23, 0x28, 0x2D, 0x32,
                                           0x37, 0x3C, 0x41, 0x46, 0x4B, 0x50, 0x5A, 0x64};
```

I can adjust `libdmdutil` accordingly, I would just need to know the level values or formula, etc.

**Comment by francisdb** [comment]
@vbousquet ping?
